corehq/apps/ota/views.py

```diff
@@ -416,6 +417,11 @@ def heartbeat(request, domain, app_build_id):
 
     if _should_force_log_submission(request):
         info['force_logs'] = True
+
+    # Select % of app users to report app integrity to PersonalID server
+    if (deterministic_random(request.couch_user.user_id) * 100) <= settings.APP_INTEGRITY_SAMPLE_PERCENT:
```

Comment:
I think this wants to include the date in the seed, otherwise it will just hit 1% of users rather than testing a all users but no more than one percent per day.

Reply:
Good catch! See https://github.com/dimagi/commcare-hq/pull/36737/commits/0e874a2fa7ca2a2875c2ad07427d157ccee54854